### PR TITLE
Qbot rules

### PR DESCRIPTION
**Pull request by nasbench**
New rules inspired by this report https://www.elastic.co/security-labs/exploring-the-qbot-attack-pattern

### CODE REVIEW

File: rules/windows/process_creation/proc_creation_win_schtasks_system.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    selection_run:
+        CommandLine|contains: '/ru '
+    selection_user:
+        CommandLine|contains:
+            - 'NT AUT'
+            - ' SYSTEM '
+    filter:
+        # FP from test set in SIGMA
+        ParentImage|contains|all:
+            - '\AppData\\Local\Temp\'
```
Comment (frack113) [inline rules/windows/process_creation/proc_creation_win_schtasks_system.yml:27]: Not only 1 `\` : '\AppData\Local\Temp\' ?

File: rules/windows/process_creation/proc_creation_win_susp_schtasks_change.yml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+            - 'certutil'
+            - 'bitsadmin'
+            - 'bash.exe'
+            - 'bash '
+            - 'scrcons'
+            - 'wmic '
+            - 'wmic.exe'
+            - 'forfiles'
+            - 'scriptrunner'
+            - 'hh.exe'
```
Comment (frack113) [inline rules/windows/process_creation/proc_creation_win_susp_schtasks_change.yml:58]: `hh ` works too.

File: rules/windows/process_creation/proc_creation_win_schtasks_system.yml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        Image|endswith: '\schtasks.exe'
+        CommandLine|contains:
+            - ' /change '
+            - ' /create '
+    selection_run:
+        CommandLine|contains: '/ru '
+    selection_user:
+        CommandLine|contains:
+            - 'NT AUT'
+            - ' SYSTEM '
```
Comment (frack113) [inline rules/windows/process_creation/proc_creation_win_schtasks_system.yml:23]: ' SYSTEM ' more '\SYSTEM' ?

Comment (Neo23x0) [reply rules/windows/process_creation/proc_creation_win_schtasks_system.yml:23]: Let's use our good old:
```
        User|contains: 
            - 'AUTHORI'
            - 'AUTORI'
```

Comment (nasbench) [reply rules/windows/process_creation/proc_creation_win_schtasks_system.yml:23]: "SYSTEM" is a valid value for "schtasks" which is equivalent to "NT AUTHORITY\SYSTEM" and I didn't want to use "AUTORI" in the command-line as it contains folders and username and "AUTORI" cannot be as specific in my opinion

**Pull request merged**